src/vorta/i18n/__init__.py

```diff
@@ -13,7 +13,7 @@ def init_translations(app):
     global application, translator, locale  # if we don't keep a reference on these, it stops working. pyqt bug?
     application = app
     translator = QTranslator()
-    locale = QLocale()
+    locale = QLocale('de')
```

Comment:
used for debugging and forget to remove again?

Reply:
i used this to debug:
```
LANG=de_DE vorta --foreground
```

Reply:
Yepp. Will use the env-var from now.